helidon-maven-plugin/src/it/projects/report-test1/pom.xml

```diff
@@ -0,0 +1,62 @@
+<!--
+
+    Copyright (c) 2020 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.helidon.build-tools.report.tests</groupId>
+    <artifactId>report-test1</artifactId>
+    <version>@project.version@</version>
+    <name>Test Report Goal</name>
+    <packaging>pom</packaging>
+
+    <properties>
+        <helidon.version>2.0.1</helidon.version>
```

Comment:
Do you expect this version to be updated as part of the release process? If not I'd prefer to have it hard-coded to avoid confusion.

Reply:
Fixed. This is not meant to be updated as part of a release.